### PR TITLE
Adding devices to Matter: add prerequisites

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Adding devices to Matter: update
- Add prerequisites
- update procedures on adding via iPhone or Android
- ToDo: this PR is mainly about adding the prerequisites and some minor updates of the basic procedures on adding a new device.
- The update of the other procedures and adding additional content are topics for later PRs.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/matter.markdown [hunk 1]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
 
-## Adding Matter devices to Home Assistant
+## Adding a Matter device to Home Assistant
 
 Each Matter network is called a fabric. Each home automation controller that controls Matter devices has its own "fabric". You can add devices directly to the fabric of your Home Assistant instance, or share them from another fabric (ie Google, Apple) to Home Assistant's fabric. We're going to explore all these options below.
 
-### Add a device using the iOS Companion app
+### Prerequisites
+
+- On the device packaging, check for the Matter logo and for either the Wi-Fi or the Thread logo.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
- On the device packaging, check for both the Matter logo and for either the Wi-Fi or the Thread logo.
```

File: source/_integrations/matter.markdown [hunk 2]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+- Check if the QR code is only on the packaging or if it is also on the device.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store it in a save place.
+  - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown:110]: Note that almost every device has it also printed on the device, and some even laser cut it on the device (so it wont fade over time). Also there are a few exceptions known, such as WiZ lights, where you enable Matter support for their app and you get the QR/code generated from within their app, the same as Matter bridges.

Not sure if its worthwhile mentioning that here?

File: source/_integrations/matter.markdown [hunk 3]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+- In Home Assistant, have the Matter integration installed.
+  - Go to {% my integrations title="**Settings** > **Devices & services**" %}.
+  - Add the **Matter (BETA)** integration.
+  - When prompted to **Select the connection method**:
+    - If you run Home Assistant OS in a regular setup: select **Submit**.
+      - This will install the official Matter server add-on.
+    - If you are already running the Matter server in another add-on, a custom container, or natively:
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
    - If you are already running the Matter server in another add-on, or a custom container:
```

Comment (marcelveldt) [reply source/_integrations/matter.markdown]: The Matter server cant be run other than add-on (recommended) or container (only if you are technically experienced)

File: source/_integrations/matter.markdown [hunk 4]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+      - Deselect the checkbox, then Select **Submit**.
+      - In the next step, provide the URL to your Matter server.
+- Have either an Android or iPhone ready and Bluetooth enabled. For information why Bluetooth is required, refer to the section on [Bluetooth used during commissioning](#bluetooth-used-during-commissioning):
+  - Android:
+    - Have an Android phone (a full Android, not F-Droid).
+    - Have the Home Assistant Companion app installed.
+    - Have Google Home app installed on the Android.
+    - We are not going to add the new device to Google Home. The app is needed because Google included the Matter SDK there.
+    - If you are using Thread: Make sure there is a Thread border router device (Nest Hub v2 or Nest Wi-Fi Pro) added to Home Assistant.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
    - If you are using Thread: Make sure there is a Thread border router device (Nest Hub v2 or Nest Wi-Fi Pro) present in your home network.
```

File: source/_integrations/matter.markdown [hunk 5]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+  - iPhone
+    - Version 16 or higher
+    - If you are using Thread: Make sure there is a Thread border router device (HomePod Mini or V2, Apple TV 4K) added to Home Assistant.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
    - If you are using Thread: Make sure there is a Thread border router device (HomePod Mini or V2, Apple TV 4K) present in your home network.
```

File: source/_integrations/matter.markdown [hunk 6]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+      - For instruction on how to add these devices to Home Assistant, refer to the instructions on [adding a HomeKit device](/integrations/homekit_controller/#adding-a-homekit-device-via-ethernet-or-wi-fi).
+
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
```

Comment (marcelveldt) [reply source/_integrations/matter.markdown]: Maybe we should link here to the Thread documentation or any other documentation how thread border routers work and how we utilize them ?

File: source/_integrations/matter.markdown [hunk 7]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+- If you are using Wi-Fi: Most Matter devices use the 2.4&nbsp;GHz frequency for Wi-Fi. For this reason, make sure your phone is in the same 2.4&nbsp;GHz network where you want to operate your devices.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
- If you are adding a Wi-Fi based Matter device: Matter devices use the 2.4&nbsp;GHz frequency for Wi-Fi. For this reason, make sure your phone is in the same 2.4&nbsp;GHz network where you want to operate your devices.
```

File: source/_integrations/matter.markdown [hunk 8]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
 
 1. Open The Home Assistant app on your phone.
-2. Go to {% my integrations title="**Settings** > **Devices & Services**" %}.
-3. On the **Devices** tab, press the **Add device** button.
-4. Choose **Add Matter device** as the top of the list.
-5. Scan the QR-code of the Matter device with your phones camera or select the **Setup without QR-code** button to manually enter the commission code.
-6. The process will start adding the device which takes up to a few minutes.
-7. If you're adding a test board (e.g. ESP32 running the example apps) and commissioning fails, you might need to take some actions in the Google Developer console, have a look at any instructions for your test device.
-8. Once the process is complete and you pressed the **Done** button, you are redirected to the device within Home Assistant. It is ready for use.
+2. Power up the device by plugging it in or add a battery. Most devices will no go into pairing mode.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
2. Power up the device by plugging it in or add a battery. Most devices will now go into pairing mode.
```

File: source/_integrations/matter.markdown [hunk 9]
```diff
@@ -98,38 +98,85 @@ If you run Home Assistant in a container, you can run a Docker image of the [Mat
...
+4. If you did not see a pop-up, go to {% my integrations title="**Settings** > **Devices & Services**" %}.
+   - On the **Devices** tab, select the **Add device** button.
+   - Select **Add Matter device**.
+   - Scan the QR-code of the Matter device with your phone camera or select **Setup without QR-code** to manually enter the commission code.
+      - This starts the commissioning process which may take a few minutes.
+   - If you're adding a test board (e.g. ESP32 running the example apps) and commissioning fails, you might need to take some actions in the Google Developer console, have a look at any instructions for your test device.
+   - Once the process is complete, select **Done**.
+5. To view the device details, go to {% my integrations title="**Settings** > **Devices & Services**" %} and select the **Matter** integration.
+6. By default, the device is called **Matter**. To rename it, select the three-dots and **Rename** the device.
+   ![image](/images/integrations/matter/matter-android-rename.png)
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
6. By default, the device gets a factory specified name. To rename it, select the three-dots and **Rename** the device.
   ![image](/images/integrations/matter/matter-android-rename.png)
```

Comment (marcelveldt) [reply source/_integrations/matter.markdown]: A new device is not called Matter but just whatever has been determined for the device. Default/common is a combination of the Manufacturer name and model name but some devices have a custom name defined or even fallback to just the device type.

Comment (marcelveldt) [reply source/_integrations/matter.markdown]: The screenshot is from renaming the integration (instance), which is always just Matter.

File: source/_integrations/matter.markdown (hunk 6 above)
Comment (c0ffeeca7) [reply source/_integrations/matter.markdown]: Agreed. I was thinking about that, too. Can we add the links once we double-checked/updated the Thread docs? I want to be sure they are updated before linking there. Making a note for another PR.

File: source/_integrations/matter.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/_integrations/matter.markdown:110]: I would prefer to keep it short here. The most important is to keep a copy of the code when it is only on the packaging. The ones from the app and from the device can be retrieved. But if it's only on the packaging, finding that code again might be impossible.

**Review by marcelveldt — APPROVED** [review]
thanks!

**Pull request merged**